mteb/tasks/Reranking/ara/NamaaMrTydiReranking.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+
+class NamaaMrTydiReranking(AbsTaskReranking):
+    metadata = TaskMetadata(
+        name="NamaaMrTydiReranking",
+        description="MrTydi reranking dataset for arabic reranking evaluation",
+        reference="https://huggingface.co/NAMAA-Space",
+        dataset={
+            "path": "NAMAA-Space/mteb-eval-mrtydi",
+            "revision": "bb3638ffe3b2be76fe2e5a4581123923afee5cda",
+        },
+        type="Reranking",
+        category="s2s",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["ara-Arab"],
+        main_score="map",
+        date=("2024-12-09", "2024-12-09"),
+        domains=[],
+        task_subtypes=[],
+        license="not specified",
+        annotations_creators="human-annotated",
+        dialect=[],
+        sample_creation="created",
+        prompt="",
+        bibtex_citation="",
+    )
+
+    def load_data(self, **kwargs):
```

Comment:
@Samoed did we implement a way to avoid overwriting the load_data function here? (I seem to recall that we added that, but I can't seem to find it)

Reply:
Reranking datasets can be easily loaded. The only issue I see in the code is that the default split is specified, but it should probably work without any changes to `load_data`

Reply:
Perfect - @omarelshehy will you then avoid overwriting the load_data(), though you are free to use overwrite the dataset_transform() if required.

Reply:
@KennethEnevoldsen @Samoed I will work on it, and all other requirements of course. Thanks a lot for the feedback